foreign_cc/private/framework.bzl

```
@@ -452,6 +452,10 @@ def cc_external_rule_impl(ctx, attrs):
         use_default_shell_env = "win" in os_name(ctx),
         # this is ignored if use_default_shell_env = True
         env = env,
+        progress_message = "Building {target_name} (Foreign Cc - {configure_name})".format(
```

Can we actually prefix the progress message with the Foreign Cc tag?
```suggestion
        progress_message = "Foreign Cc - {configure_name}: Building {target_name}".format(
```
I find it easier to see what's going on at a glance this way.

---

> Can we actually prefix the progress message with the Foreign Cc tag?
> 
> I find it easier to see what's going on at a glance this way.

Sure, done